spec/02-integration/02-cmd/02-start_stop_spec.lua

```diff
@@ -39,6 +41,102 @@ describe("kong start/stop", function()
     assert(helpers.kong_exec("start --conf " .. helpers.test_conf_path))
     assert.truthy(helpers.path.exists(helpers.test_conf.kong_env))
   end)
+  it("start without daemonizing will properly log to stdout/stderr", function()
+    --[[
+    What are we testing exactly?
+    When Kong starts, the cli commands will shell out to start the 'nginx'
+    executable. The stdout/stderr of this command is redirected to be captured
+    in case of errors.
+    A problem arises when Kong runs in the foreground 'nginx_daemon=off', and
+    the nginx log output is set to '/dev/stderr' and '/std/stdout'.
+    In this case the shell command to start Kong will capture all output send
+    to stdout/stderr by nginx. Which causes the tempfiles to grow uncontrolable.
+
+    So when Kong is run in the foreground, no redirects should be used when
+    starting Kong to prevent the above from happening. Or in other words we
+    want the stdout/stderr of the "kong start" command to receive all output
+    of the nginx logs. Instead of them being swallowed by the intermediary
+    nginx start command (used by "kong start" under the hood).
+    --]]
+    local xos = _G.os  -- trick the linter here in accepting global updates
```

Comment:
instead of tricking LuaCheck, you can use the special comment `-- luacheck: ignore` (or preferrably a more specific one as specified here: http://luacheck.readthedocs.io/en/stable/inline.html )

Reply:
k thx, will look into it

biggest issue for now is the test not passing CI. Needs a better way, ideas anyone?

Reply:
This week @kikito mentioned issues with creating temporary files on Travis, but I never experienced that myself (one of my tests does create a temp file). Any idea on what's causing it to fail?

Reply:
hmmm looking at it again, it changed. Initially it timedout while waiting for Kong to start, but now also seeing other errors. Probably on the merged branch, since the config properties changed.

I'll rebase first